configs/defs/rocky.textproto

```diff
@@ -0,0 +1,1179 @@
+      "       }"
+      "      }"
+      "      existence:{should_exist:true}"
+      "    }"
+      "}}"
+  }
+}
+benchmark_configs: {
+  id: "journald-default-file-permissions-configured"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure journald default file permissions configured"
+    description:
+      "Journald will create logfiles that do not already existon the system. This setting controls "
+      "what permissions will be applied to these newly created files."
+    rationale:
+      "It isimportantto ensure that log files have the correct permissions to ensure that sensitive "
+      "data is archived and protected."
+    remediation:
+      "If the default configuration is not appropriate for the site specific requirements, copy "
+      "/usr/lib/tmpfiles.d/systemd.conf to /etc/tmpfiles.d/systemd.conf and modifyas "
+      "required. Requirements is either 0640 or site policy if that is less restrictive."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "        path:\"/usr/lib/tmpfiles.d/systemd.conf\""
+      "       }"
+      "      }"
+      "    permission:{"
+      "      set_bits: 0640"
+      "      user:{"
+      "        name: \"root\""
+      "        should_own: true"
+      "      }"
+      "    }"
+      "    }"
+      "  }"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "        path:\"/usr/lib/tmpfiles.d/systemd.conf\""
+      "       }"
+      "      }"
+      "    existence:{should_exist: false}"
+      "  }"
+      "}}"
+  }
+}
+benchmark_configs: {
+  id: "systemwide-crypto-policy-is-not-overridden"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure system-wide crypto policy is not over-ridden"
+    description:
+      "System-wide Crypto policy can be over-ridden or opted out of for openSSH"
+    rationale:
+      "Over-riding or opting out of the system-wide crypto policy could allow for the use of less "
+      "secure Ciphers, MACs, KexAlgorithms and GSSAPIKexAlgorithm"
+    remediation:
+      "Run the following commands:\n"
+      "# sed -ri \"s/\"\\s* (CRYPTO_POLICY\\s*=.*)$/# \\1/\" /etc/sysconfig/sshd\n"
+      "# systemctl reload sshd"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/sysconfig/sshd\"}}"
+      "      content_entry:{"
+      "        match_type: NONE_MATCH"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*CRYPTO POLICY=\""
```

Comment:
Filter regexes implicitly include `^` and `$`: https://github.com/google/localtoast/blob/master/scannerlib/configchecks/content_entry_file_check.go#L80
So this would only match if there's nothing after the `=`.
Add `.*` before and after.